packages/react-theme-provider/README.md

```diff
@@ -0,0 +1,66 @@
+# @fluentui/react-theme-provider
+
+**React theming component and hook for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+
+# Installation
+
+```bash
+yarn add @fluentui/react-theme-provider
+```
+
+# Example usage
+
+First, ensure you use an existing theme, or create your own. Example:
+
+```js
+/**
+ * Note: Themes are partial by default. They will become fully qualified
+export const theme: Theme = {
+  /* Provide any stylesheets which should come along with the theme */
+  stylesheets: [
+    '.className { ... }',
+    ...
+  ],
+
+  /* Provide standard fluent tokens here. */
+  tokens: {
+
+    body: {
+      fill: {}
+    },
+
+    site: {
+
+    }
+  }
+};
+```
+
+Use the theme with Fluent UI by wrapping content within the provider:
+
+```tsx
+import { ThemeProvider } from '@fluentui/react-theme-provider';
+import { theme } from './theme';
+
+export const App = () => (
+  <ThemeProvider theme={theme}>
+    <>...app</>
+  </ThemeProvider>
+);
+```
+
+## Applying the theme to the body element
```

Comment:
1. this is not supported in your change
2. why is this feature needed?

Reply:
I've removed the comment. However I believe it will be needed. Otherwise you won't have a way to apply the theme to the body.